src/main.ts

```diff
@@ -9,7 +9,7 @@ import { ConstantStore } from "./stores/constant.store";
 
 async function main() {
     try {
-        await initSentry();
```

Comment:
We'd like to be notified of errors that can possibly occur whenever the peer is being initialized. We should, therefore, wait before sentry has been initialized.

Reply:
The initSentry is not an async function and thus should not be awaited. (This was also flagged by sonarqube)